src/webapp/pages/settings/CreateButton.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback } from "react";
+import { Description, OpenInBrowser } from "@material-ui/icons";
+import { SpeedDial, SpeedDialAction, SpeedDialIcon } from "@material-ui/lab";
+
+import i18n from "../../../utils/i18n";
+import { useAppContext } from "../../contexts/app-context";
+
+export interface CreateButtonProps {
+    onAddLandingPage: () => void;
+}
+
+export const CreateButton: React.FC<CreateButtonProps> = props => {
+    const { onAddLandingPage } = props;
+
+    const { setAppState } = useAppContext();
+    const onAddModule = useCallback(() => {
+        setAppState({ type: "CREATE_MODULE" });
+    }, [setAppState]);
+
+    const [isOpen, open] = useOpenAction();
+
+    return (
+        <>
+            <SpeedDial
+                open={isOpen}
+                onClick={open}
+                style={styles.speedDial}
+                ariaLabel={i18n.t("Create")}
+                icon={<SpeedDialIcon />}
+            >
+                <SpeedDialAction
+                    onClick={onAddModule}
+                    tooltipOpen
+                    icon={<OpenInBrowser />}
+                    tooltipTitle={i18n.t("Module")}
+                />
+
+                <SpeedDialAction
+                    onClick={onAddLandingPage}
+                    tooltipOpen
+                    icon={<Description />}
+                    tooltipTitle={i18n.t("Landing Page")}
+                />
+            </SpeedDial>
+        </>
+    );
+};
+
+const styles = {
+    speedDial: { position: "fixed" as const, bottom: 16, right: 16 },
+};
```

Comment:
To avoid having free type here (and needing as const) with no validations, we could use satisfies Record<string, React.CSSProperties>; But that's TS 4.4, and in any case it's good to have abstractions, so my proposal:

```ts
const styles = cssStyles({
    speedDial: { position: "fixed", bottom: 16, right: 16 },
});

function cssStyles<T extends Record<string, React.CSSProperties>>(styles: T): T {
    return styles;
}
```